boards/st/stm32u5g9j_dk2/doc/index.rst

```diff
@@ -0,0 +1,176 @@
+.. zephyr:board:: stm32u5a9j_dk
+
+Overview
+********
+
+The STM32U5G9J-DK2 Discovery kit is a complete demonstration and development
+platform for the STM32U5G9ZJT6Q microcontroller, featuring an Arm|reg| Cortex|reg|‑M33
+core with Arm|reg| TrustZone|reg|.
+
+Leveraging the innovative ultra-low power-oriented features, 3 Mbytes of
+embedded SRAM, 4 Mbytes of embedded flash memory, and rich graphics features,
+the STM32U5G9J-DK2 Discovery kit enables users to prototype applications
+with state-of-the-art energy efficiency, as well as providing stunning and
+optimized graphics rendering with the support of a 2.5D Neo-Chrom accelerator,
+chrom-ART Accelerator, and Chrom-GRC™ MMU.
+
+The STM32U5G9J-DK2 Discovery kit integrates a full range of hardware features
+that help the user evaluate all the peripherals, such as a 5" RGB 800x480 pixels
+TFT colored LCD module with a 24‑bit RGB interface and capacitive touch panel,
+USB Type-C|reg| HS, Octo‑SPI flash memory device, ARDUINO|reg|, and STLINK-V3EC
+(USART console).
+
+The STM32U5G9J-DK2 Discovery kit integrates an STLINK-V3EC embedded in-circuit
+debugger and programmer for the STM32 microcontroller with a USB Virtual COM
+port bridge and comes with the STM32CubeU5 MCU Package, which provides an STM32
+comprehensive software HAL library as well as various software examples.
+
+.. image:: img/top_view.webp
+     :align: center
+     :alt: STM32U5G9J-DK2 Top View
+
+.. image:: img/bottom_view.webp
+     :align: center
+     :alt: STM32U5G9J-DK2 Bottom View
+
+More information about the board can be found at the `STM32U5G9J-DK2K website`_.
+More information about STM32U5G9ZJT6Q can be found here:
+
+- `STM32U5G9ZJ on www.st.com`_
+- `STM32U5 Series reference manual`_
+- `STM32U5Gxxx datasheet`_
+
+Supported Features
+==================
+
+The current Zephyr stm32u5g9j_dk2 board configuration supports the following
+hardware features:
+
+.. zephyr:board-supported-hw::
+
+Other hardware features have not been enabled yet for this board.
+
+The default configuration per core can be found in the defconfig file:
+:zephyr_file:`boards/st/stm32u5g9j_dk2/stm32u5g9j_dk2_defconfig`
```

Comment:
Just keep `.. zephyr:board-supported-hw::`. All the rest can be removed, to keep it consistent with other board documentation